src/coreclr/jit/hwintrinsiccodegenarm64.cpp

```diff
@@ -2252,6 +2253,24 @@ void CodeGen::genHWIntrinsic(GenTreeHWIntrinsic* node)
                 break;
             }
 
+            case NI_Sve_LoadVectorFirstFaulting:
+            {
+                insScalableOpts sopt = INS_SCALABLE_OPTS_LSL_N;
+                if (opt == INS_OPTS_SCALABLE_B)
+                {
+                    sopt = INS_SCALABLE_OPTS_NONE;
+                }
+                GetEmitter()->emitIns_R_R_R_R(ins, emitSize, targetReg, op1Reg, op2Reg, REG_ZR, opt, sopt);
```

Comment:
did you verify where do we set the `LSL #1` vs. `LSL #2`, etc?

Reply:
It's part of the encoding for the instruction, they will always be that. We verified it already when we implemented the instructions.